pallets/fiat_filter/src/lib.rs

```diff
@@ -144,8 +145,11 @@ impl<T: Config> Module<T> {
         };
         match call.is_sub_type() {
             Some(attest::Call::set_claim(_attester, attestation, _sig)) => {
-                let iri_size_100bytes: u32 =
-                    (attestation.iri.as_ref().unwrap_or(&[1].to_vec()).len() as u32).div_ceil(100);
+                let iri_size_100bytes: u32 = attestation
+                    .iri
+                    .as_ref()
+                    .map_or_else(|| 1, |i| (i.len() as u32))
```

Comment:
My comment here https://github.com/docknetwork/dock-substrate/pull/97#discussion_r607158791 had `div_ceil(100)` in the callback to `map_or_else` so that you use `div_ceil` only when you have a value in the Option but you have `div_ceil` applied to the result of `map_or_else` which will use `div_ceil` regardless of the contents of the Option which is same as what you had before.